crd-convert/src/test/java/io/strimzi/kafka/crd/convert/converter/KafkaConverterTestBase.java

```diff
@@ -0,0 +1,492 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.crd.convert.converter;
+
+import io.fabric8.kubernetes.api.model.AffinityBuilder;
+import io.fabric8.kubernetes.api.model.TolerationBuilder;
+import io.strimzi.api.annotations.ApiVersion;
+import io.strimzi.api.kafka.model.Kafka;
+import io.strimzi.api.kafka.model.KafkaBuilder;
+import io.strimzi.api.kafka.model.KafkaClusterSpec;
+import io.strimzi.api.kafka.model.listener.arraylistener.KafkaListenerType;
+import io.strimzi.api.kafka.model.template.ExternalTrafficPolicy;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+
+abstract class KafkaConverterTestBase {
```

Comment:
A sentence of javadoc explaining how this ABC is supposed to be used wouldn't hurt.